github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -255,6 +250,7 @@ presubmits:
       preset-shared-images: "true"
     max_concurrency: 11
     name: pull-kubevirt-e2e-k8s-1.23-sig-storage-cgroupsv2
+    run_if_changed: pkg/virt-handler/cgroup/.*|pkg/virt-handler/hotplug-disk/.*|pkg/hotplug-disk/.*|cmd/virt-chroot/cgroup.go
```

Comment:
Please also add `optional: false` as there was
else it adds an implicit `optional: true`

same here
https://github.com/kubevirt/project-infra/pull/2188/files#diff-26f7cd0389085c1da47f919a02c4dbeca5ddd96ca00d8e49c0862f698475aaf9R212

Reply:
As said before, `optional: false` is the default, and default values will always be removed when serializing the configuration to disk, which is what the automation does.

Sane for `skip_report: false`

Reply:
From what k8s community told me `optional: true` is the default, unless `always_run: true` is used

Reply:
I don't see any reference in the codebase that sets `Optional` to `true`. Please refer me to the discussion or to the code ICIMI.

As said before, if the [serialization removes it as stated here](https://github.com/kubernetes/test-infra/blob/20e880da9b16b3237bf0ddbee2787c05f285eefd/prow/config/jobs.go#L149) I'd prefer to leave it like that, as the automation will remove the change again anyway.

Reply:
(sent you link offline)

Reply:
:facepalm: I totally missed that this is not about the standard sig lanes, but about the cgroupsv2 lanes. Thanks, good catch!

Reply:
Also you might want to update the cgroupsv2 to 1.24 as well ? (there are two)

Reply:
And skip_report is definitely false from what I remember.

Reply:
> To reiterate about this, IIRC if run_if_changed is set, the [job will be required to pass by tide if the run_if_changed matches the PR](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#requiring-jobs-for-auto-merge-through-tide).
> 
> So IIUC no need to set this explicitly here.

You might be right, best if we can validate this if possible please
note that we won't see "Required" on the job list on the PR bottom, but we should see "Required = true" at the table in case of a failure of that job

Reply:
IIRC while the job will not be stated as required (which strictly seen it isn't) tide will still complain about the failing job, should the run_if_changed match. IMHO while looking weirt that's the correct thing to show.